platforms/web/src/utils/accessibility.ts

```diff
@@ -0,0 +1,20 @@
+export const attachAccessibilityListener = () => {
+  let isTabbing = false;
+
+  // when the user presses the Tab key, we add a `is-tabbing` className to the body element, so we can enable outline
```

Comment:
What kind of outline do we use here?

Reply:
The outline mixins can be found here:

https://github.com/jwplayer/ott-web-app/pull/451/files#diff-6e7f14d8119a5c1f12eddb73c5dcace8e673123355c88285acea294926f03011R27-R39

Reply:
Nice, thank you!